resolver.go

```diff
@@ -17,6 +17,17 @@ import (
 	"github.com/xeipuuv/gojsonschema"
 )
 
+// Locations for input parameters. These are used in struct field tags to
+// specify the location from which the parameter value gets set. It is also
+// used to generate JSON Path locations for error reporting. For example,
+// `path.id` or `body.foo.bar[0].baz` might have validation errors.
+const (
+	locationPath   = string(inPath)
```

Comment:
Silly question: where is `inPath` beingn defined? I don't see it in the scope of what's changed. I'm sure it's being set; I'm just not sure where.

Reply:
It's part of the internal OpenAPI parameter type definition in `openapi.go` but we don't just reuse it because those are limited to non-body params. Here we cast to strings for convenience and include the `body` one for use below. I could have just hard-coded the strings again but figured we can re-use the OpenAPI names since they are the same.